.evg.yml

```diff
@@ -102,9 +103,124 @@ tasks:
+    commands:
+      - func: "pull augmented SBOM from Silk"
+      - func: "publish augmented SBOM"
+
 functions:
+  "push SBOM Lite to Silk":
+    - command: ec2.assume_role
+      params:
+        role_arn: ${assume_role_arn}
+        duration_seconds: 3600
+    - command: s3.get
+      params:
+        aws_key: ${AWS_ACCESS_KEY_ID}
+        aws_secret: ${AWS_SECRET_ACCESS_KEY}
+        aws_session_token: ${AWS_SESSION_TOKEN}
+        local_file: artifacts/ssdlc/${SBOM_LITE_NAME}
+        remote_file: artifacts/${version_id}/ssdlc/${SBOM_LITE_NAME}
+        content_type: application/json
+        bucket: evg-bucket-mongo-jdbc-driver
+    - command: shell.exec
+      type: test
+      params:
+        shell: bash
+        working_dir: mongo-jdbc-driver
+        script: |
+          ${PREPARE_SHELL}
+          cat << EOF > silkbomb.env
+          SILK_CLIENT_ID=${SILK_CLIENT_ID}
+          SILK_CLIENT_SECRET=${SILK_CLIENT_SECRET}
+          EOF
+
+          ls -lrt $SSDLC_DIR
+
+          echo "SBOM_LITE_NAME = $SBOM_LITE_NAME"
+
+          echo "-- Uploading initial SBOM Lite to Silk --"
+          docker run -i --platform="linux/amd64" --rm -v "$PWD":/pwd \
+          --env-file silkbomb.env \
+          artifactory.corp.mongodb.com/release-tools-container-registry-public-local/silkbomb:1.0 \
+          upload --silk-asset-group ${SILK_ASSET_GROUP} --sbom-in /pwd/artifacts/ssdlc/$SBOM_LITE_NAME
+          echo "-------------------------------"
+
+  "pull augmented SBOM from Silk":
+    - command: ec2.assume_role
+      params:
+        role_arn: ${assume_role_arn}
+        duration_seconds: 3600
+    - command: shell.exec
+      type: test
+      params:
+        shell: bash
+        working_dir: mongo-jdbc-driver
+        script: |
+          ${PREPARE_SHELL}
+          
+          mkdir -p $SSDLC_DIR
+
+          cat << EOF > silkbomb.env
+          SILK_CLIENT_ID=${SILK_CLIENT_ID}
+          SILK_CLIENT_SECRET=${SILK_CLIENT_SECRET}
+          EOF
+
+          echo "AUGMENTED_SBOM_NAME = $AUGMENTED_SBOM_NAME"
+
+          echo "-- Downloading augmented SBOM --"
+          docker run -i --platform="linux/amd64" --rm -v "$PWD":/pwd \
+          --env-file silkbomb.env \
+          artifactory.corp.mongodb.com/release-tools-container-registry-public-local/silkbomb:1.0 \
+          download --silk-asset-group ${SILK_ASSET_GROUP} --sbom-out /pwd/artifacts/ssdlc/$AUGMENTED_SBOM_NAME
+          echo "-------------------------------"
+
+          ls -lrt $SSDLC_DIR
+    - command: s3.put
+      params:
+        aws_key: ${AWS_ACCESS_KEY_ID}
+        aws_secret: ${AWS_SECRET_ACCESS_KEY}
+        aws_session_token: ${AWS_SESSION_TOKEN}
+        local_file: mongo-jdbc-driver/artifacts/ssdlc/${AUGMENTED_SBOM_NAME}
+        remote_file: artifacts/${version_id}/ssdlc/${AUGMENTED_SBOM_NAME}
+        content_type: application/json
+        bucket: evg-bucket-mongo-jdbc-driver
+        permissions: public-read
+
+  "publish augmented SBOM":
```

Comment:
I have been thinking about this as well on the odbc side, but wondering if we need this step? For sqlproxy, I think it makes sense, because the publish step moves from mciuploads (which we just throw things in?) into the info-mongodb-com bucket which is the public downloadable one we put download center artifacts in. 

If we plan on doing this every patch, should we just put the augmented sbom into this place in the put command of the `pull augmented SBOM from Silk` task? Or am I missing some nuance here / in sqlproxy?